src/lib.rs

```diff
@@ -115,6 +116,7 @@ use serde::{Deserialize, Deserializer, Serialize};
 mod dependency;
 pub mod diagnostic;
 mod errors;
+pub mod libtest;
```

Comment:
Since this may change in rustc with any rustc release, what do you think about adding an `unstable` cargo feature to `cargo_metadata` and hiding this module behind it?

Reply:
Seems like a good idea